packages/contracts-bedrock/src/dispute/FaultDisputeGame2.sol

```diff
@@ -942,4 +951,35 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, ISemver {
             ? Hash.wrap(keccak256(abi.encode(_disputed, _disputedPos)))
             : Hash.wrap(keccak256(abi.encode(_starting, _startingPos, _disputed, _disputedPos)));
     }
+
+    function _traceAncestorV2(Position _position, uint256 _intervalDepth)
+        internal
+        pure
+        returns (Position ancestor_)
+    {
+        if (_position.depth() % _intervalDepth != 0 || _position.depth() < _intervalDepth) {
+            revert InvalidClaim();
+        }
+        ancestor_ = _position.traceAncestor();
+        // Position 1 is not the value submitted by the Actors.
```

Comment:
I am not sure of the comment's meaning.  In addition, I do not think we need a special check for `1 == ancestor_.raw()`